public/application.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function() {
+  $("td").click(function(e) {
+      e.preventDefault();
+      $.ajax({
+          type: "PUT",
+          url: "/game",
+          data: {
+              space: $(this).attr('id'),
+          },
+          success: function(result) {
```

Comment:
instead of callbacks lets use Promises, or Async/await. callbacks are the old, old ways to handle asynchronous functions. same with error.